hawtjni-runtime/src/main/java/org/fusesource/hawtjni/runtime/Library.java

```diff
@@ -20,27 +20,36 @@
  * 
  * It will search for the library in order at the following locations:
  * <ol>
- * <li> in the custom library path: If the "<code>library.${name}.path</code>" System property is set to a directory 
+ * <li> in the custom library path: If the "<code>library.${name}.path</code>" System property is set to a directory,
+ * subdirectories are searched:
+ *   <ol>
+ *   <li> "<code>${platform}/${arch}</code>"
+ *   <li> "<code>${platform}</code>"
+ *   <li> "<code>${os}</code>"
+ *   <li> "<code></code>"
+ *   </ol>
+ *   for 2 namings of the library:
  *   <ol>
  *   <li> as "<code>${name}-${version}</code>" library name if the version can be determined.
  *   <li> as "<code>${name}</code>" library name
  *   </ol>
  * <li> system library path: This is where the JVM looks for JNI libraries by default.
  *   <ol>
+ *   <li> as "<code>${name}${bit-model}-${version}</code>" library name if the version can be determined.
```

Comment:
Why did you add that? System library path (`LD_LIBRARY_PATH`, `rpath`) will unlikely contain the bit model. It looks unnecessary to me.